cellxgene_schema_cli/cellxgene_schema/migrate.py

```diff
@@ -72,6 +80,40 @@ def migrate(input_file, output_file, collection_id, dataset_id):
     #   <custom transformation logic beyond scope of replace_ontology_term>
     # ...
 
+    if dataset.uns["title"] == "Time-resolved single-cell analysis of Brca1 associated mammary tumourigenesis reveals aberrant differentiation of luminal progenitors":
+        utils.replace_ontology_term(dataset.obs, "cell_type", {"CL:0000451": "CL:4047054"})
+
+    if collection_id == "126afc71-47fb-4e9d-8aaf-9d9f61e0ac77":
+        utils.map_ontology_term(
+            dataset.obs,
+            "disease",
+            "disease_group",
+            {
+                "Healthy": "PATO:0000461",
+                "HIV": "MONDO:0005109",
+                "VL_HIV": "MONDO:0005109 || MONDO:0011989",
```

Comment:
"VL" is MONDO:0005445 (visceral leishmaniasis) instead of MONDO:0011989
so that can be updated for VL_HIV, cVL_HIV, pVL_HIV